webdriverio/webdriver/examples/visual-e2e/test/specs/simple.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('My app', () => {
+    it('should look correct', async () => {
+        await browser.url('');
+        await browser.execute('/*@visual.init*/', 'My React App');
```

Comment:
I was just wondering, should we also explain how this works in comments/refer to the screener docs? Now it's just code without any explanation

Reply:
Doesn't the Screener documentation do that?

Reply:
That's true, but shouldn't our examples also explain this, because otherwise people would already use the correct commands and so on if they have read the Screener docs